students/tim_lurvey/lesson09/mailroom_oo/donor_classes.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+__author__ = 'Timothy Lurvey'
+
+from running_total import RunningTotal
+
+
+class Donor(RunningTotal):
```

Comment:
there isn't a good use case here to have Donor inherit from another class

Reply:
Outside of class, I repurposed the runningTotal object in my logging decorator that tracks how many times each method is called and how much time they have accumulated.  It's much better than the dictionary, even if it is overkill.